src/Router.js

```diff
@@ -1,24 +1,37 @@
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Outlet } from 'react-router-dom';
```

Comment:
동일한 패키지인 react-router-dom에서 import하고 있기 때문에 함께 import하시면 됩니다.
```suggestion
import { BrowserRouter, Routes, Route, Outlet } from 'react-router-dom';
```